internal/nginx/config/servers_test.go

```diff
@@ -338,7 +369,8 @@ func TestCreateServers(t *testing.T) {
 
 	cafePathRules := []dataplane.PathRule{
 		{
-			Path: "/",
+			Path:     "/",
+			PathType: string(v1beta1.PathMatchPathPrefix),
```

Comment:
would it make sense to define our own constants in the data plane changes? this way, if we change PathType implementation, we will not have to change the constants here.

Reply:
Do you mean defining constant `string` that equal the v1beta1 types?

Reply:
> Do you mean defining constant string that equal the v1beta1 types?

yep, so there is no dependency on them in the config package.

Reply:
Where do you suppose the best place to define these constants is?